configure.ac

```diff
@@ -4,12 +4,12 @@ AC_PREREQ([2.69])
 
 
 m4_define([MAJOR_VERSION], 1)
-m4_define([MINOR_VERSION], 0)
+m4_define([MINOR_VERSION], 1)
 m4_define([MICRO_VERSION], 0)
 
 AC_INIT([ledmon],[MAJOR_VERSION.MINOR_VERSION.MICRO_VERSION])
 AC_CONFIG_MACRO_DIR([m4])
-AC_SUBST([PACKAGE_DATE], "March 2024")
+AC_SUBST([PACKAGE_DATE], "October 2024")
```

Comment:
...same here.

Reply:
Lgtm.

Reply:
@mtkaczyk I see this is used only in `doc` makefile. Can we remove this?

Reply:
I don't have nothing strong against so it is fine. Just keep year.